tests/python_package_test/test_dask.py

```diff
@@ -0,0 +1,210 @@
+import os
+import sys
+
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import pytest
+import scipy.sparse
+from dask.array.utils import assert_eq
+from dask_ml.metrics import accuracy_score, r2_score
+from distributed.utils_test import client, cluster_fixture, gen_cluster, loop  # noqa
+from sklearn.datasets import make_blobs, make_regression
+
+import lightgbm
+import lightgbm.dask as dlgbm
+
+data_output = ['array', 'scipy_csr_matrix', 'dataframe']
+data_centers = [[[-4, -4], [4, 4]], [[-4, -4], [4, 4], [-4, 4]]]
+
+pytestmark = [
+    pytest.mark.skipif(sys.platform != "linux", reason="Only linux is currently supported"),
```

Comment:
Is it true that only Linux is supported? Or is it "not Windows"?

I don't recall seeing any issues in the CI jobs on Mac. Could this check and skip message be changed?

```suggestion
    pytest.mark.skipif(sys.platform.startswith("win"), reason="lightgbm.dask is not currently supported on Windows"),
```

Reply:
I had problems with osx builds where the build timed-out after 1hour. I am not entirely sure what caused the problems. So for now, I suggest to support only linux and solve the issue with osx in different PR.

Reply:
oh ok no problem! I'm ok with that suggestion. I just didn't remember it.